Rubberduck.Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -13,6 +13,8 @@
 using Rubberduck.Parsing.Inspections.Resources;
 using Rubberduck.Parsing.Rewriter;
 using Rubberduck.Parsing.VBA;
+using static Rubberduck.Parsing.Grammar.VBAParser;
+using System.Diagnostics;
```

Comment:
not sure why that is included?

Reply:
`Rubberduck.Parsing.Grammar.VBAParser` supports use/compiling of references to `EndOfStatementContext`, `AgListContext`, and `AsTypeClauseContext` .  There's a Debug.Assert statement used (which could be removed if needed)  - so `System.Diagnostics`.  Is there a better, more preferred approach?

Reply:
explicitly checking for your precondition.

Reply:
@Vogel612 - TBH I'm not sure I understand your last comment.  Made some changes consistent with what I could find on the web after searching for "checking preconditions".  Hopefully the latest change is something close to what you are recommending.